internal/app.go

```diff
@@ -174,6 +179,16 @@ OPTIONS:
   {{range $index, $option := .VisibleFlags}}{{if $index}}
   {{end}}{{$option}}{{end}}{{end}}
 `
+	cli.VersionPrinter = func(c *cli.Context) {
+		db.Init(utils.DefaultCacheDir())
```

Comment:
We need to take `-cache-dir` option.

Reply:
Fixed 0952d709b17878654fad853c46de616984c4b2a5